src/api/controllers/profileController.js

```diff
@@ -118,9 +118,8 @@ export default class ProfileController {
    * @return {Object} The response Object
```

Comment:
Why do you have this file in your PR @murediane ?

Reply:
I was fixing the test that was failing